Source/WebCore/platform/graphics/mac/controls/ControlFactoryMac.h

```diff
@@ -81,6 +83,7 @@ class ControlFactoryMac final : public ControlFactoryCocoa {
     NSMenu *searchMenuTemplate() const;
     NSSliderCell *sliderCell() const;
     NSTextFieldCell *textFieldCell() const;
+    WebSwitchView *webSwitchView() const;
```

Comment:
Looking at the code again this can probably hold an NSSwitch directly. Should it be called `switchTemplate()` similar to the `NSMenu` one above?